src/com_tjvendors/admin/models/fields/modifiedby.php

```diff
@@ -13,6 +10,9 @@
 
 defined('JPATH_BASE') or die;
 
+use Joomla\CMS\Form\FormField;
+use Joomla\CMS\Factory;
+
 jimport('joomla.form.formfield');
```

Comment:
remove jimport('joomla.form.formfield'); from all field files if related class is loaded

Reply:
done